### PR TITLE
AEC Bug fixes

### PR DESCRIPTION
**Pull request by angelhof**

### CODE REVIEW

**Comment by angelhof** [comment]
@nvasilakis feel free to take a look (no need to though) and merge when you are ready :)

File: evaluation/eurosys/execute_web_index_dish_evaluation.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ while getopts 'slh' opt; do
     case $opt in
```
Comment (nvasilakis) [inline evaluation/eurosys/execute_web_index_dish_evaluation.sh:12]: The filename here contains `dish`:-)

File: evaluation/results_archive/results/test_results/micro_10_2_distr_auto_split.time [hunk 2]
```diff
@@ -0,0 +1,190 @@
+Arguments:
```
Comment (nvasilakis) [inline evaluation/results_archive/results/test_results/micro_10_2_distr_auto_split.time:1]: I am not entirely sure we need these in the repo, but I'll leave the final decision up to you:-)

**Review by nvasilakis — APPROVED** [review]

File: evaluation/results_archive/results/test_results/micro_10_2_distr_auto_split.time (hunk 2 above)
Comment (angelhof) [reply evaluation/results_archive/results/test_results/micro_10_2_distr_auto_split.time:1]: I agree that we need to have a discussion about this, for now this should OK.

**Pull request merged**